core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/shared/services/dot-edit-page-resolver/dot-edit-page-resolver.service.ts

```diff
@@ -104,4 +88,43 @@ export class DotEditPageResolver implements Resolve<DotPageRenderState> {
 
         return renderOptions;
     }
+
+    private setSite(id: string): Observable<Site> {
+        const currentSiteId = this.siteService.currentSite?.identifier;
+        const shouldSwitchSite = id && id !== currentSiteId;
+
+        // If we have a site id and is different from the current one, we switch
+        return shouldSwitchSite
+            ? this.siteService.switchSiteById(id).pipe(
+                  catchError((err: HttpErrorResponse) => {
+                      console.warn(err);
```

Comment:
use `this.dotHttpErrorManagerService.handle(err, true)` so in that way the user can know something weird happened, in an unobtrusive way (toast message)